web_src/css/base.css

```diff
@@ -344,6 +344,11 @@ a.muted,
   color: inherit;
 }
 
+button.muted {
+  background: none;
+  border: none;
+}
+
```

Comment:
I don't like to repeat myself, but that's why I said
> The effect of the muted class is what you would expect: The button loses all of its normal styling, and is defined only by its content instead.
This will help reduce a11y infractions in the future, as that was one of the major points why people didn't use <button> tags and decided on a bad fix (i.e. through <div>s) instead.

We need a class like that, no matter how it is called.
There is no way around that.
Everything else is simply not maintainable for something as large as Gitea

Reply:
I agree we need a class, but I don't agree on the muted name. I would call it `btn-link`.

Reply:
I'll push up something, it combines both `btn-link` and `muted` class.

Reply:
Pushed